src/constants/memory/wram.asm

```diff
@@ -1278,11 +1284,14 @@ wEntitiesOptions1Table::
   ds $10 ; C430 - C43F
 
 ; Entity-specific state.
+;
+; Examples:
+;  - Peahat: movement angle (0x0 = right ... 0x4 = down ... 0x8 = left ... 0xC = up)
 wEntitiesPrivateState4Table::
   ds $10 ; C440 - C44F
 
-; Number of frame before a dropped item disappears
-wEntitiesDropTimerTable::
+; Frames before the next state transition of the entity (only decremented every four frames)
+wEntitiesSlowTransitionCountdownTable::
```

Comment:
Some quick questions, before I take the time to read everything:

- It seems this is a regular countdown - so the `Table` prefix that was here initially was probably a previous mistake. `wEntitiesSlowTransitionCountdown` should be fine.
- The name could maybe be more descriptive though. Is it used for other things that droppable items?

Reply:
- I don't understand, why is that a mistake? The other countdowns have `Table` in the name, like `wEntitiesTransitionCountdownTable`. I also pretty much copied that countdown's comment.
- Yes, it's also used, for example, by Peahat in place of a normal transition countdown and by the Timer Bombite as the timer before exploding.

Reply:
> I don't understand, why is that a mistake? The other entity countdowns have Table in the name, like wEntitiesTransitionCountdownTable. I also pretty much copied that countdown's comment.

Oh, yes, that's because I started reading this PR way too late, and didn't realize it was an entity table 🤦

> Yes, it's also used, for example, by Peahat in place of a normal transition countdown and by the Timer Bombite as the timer before exploding. It seems they use it when a normal countdown would be too short.

Ha, ok, I get it.

Mmm, so I'd say we can either keep the name like this – or make the parallel with the existing `wEntitiesPrivateCountdown1Table` and `wEntitiesPrivateCountdown2Table`, and name it `wEntitiesPrivateSlowCountdownTable`. What do you think?

Reply:
Well, from what I can tell, it's usually used similarly to the regular transition countdown, which is why I went with this name. I can see that working too, though.